src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/validate/PNCIdentifier.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.personrecord.validate
+
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+
+const val LONG_PNC_ID_LENGTH = 10
+
+data class PNCIdentifier(private val inputPncId: String? = null) {
```

Comment:
Love a data class, I have missed them for the past year. Java Records look like they might be good too